lib/presentation/ui/pages/sign up/sign_up_view.dart

```diff
@@ -0,0 +1,165 @@
+                              child: Column(
+                                children: [
+                                  TextInputForm(
+                                    labelText: S.of(context).nameLabel,
+                                    hintText: S.of(context).nameHintText,
+                                    validator: NameFieldValidator(),
+                                    onFieldSaved: (value) {},
+                                    onTaped: () {},
+                                  ),
+                                  const SizedBox(height: 17),
+                                  TextInputForm(
+                                    labelText: S.of(context).emailLabel,
+                                    hintText: S.of(context).emailHintText,
+                                    validator: EmailFieldValidator(),
+                                    onFieldSaved: (value) {},
+                                    onTaped: () {},
+                                  ),
+                                  const SizedBox(height: 17),
+                                  TextInputForm(
+                                    labelText: S.of(context).passwordLabel,
+                                    hintText: S.of(context).passwordHintText,
+                                    isSensitive: true,
+                                    validator: PasswordFieldValidator(),
+                                    onChanged: (value) {
+                                      _password = value.trim();
+                                    },
+                                    onFieldSaved: (value) {},
+                                    onTaped: () {},
+                                  ),
+                                  const SizedBox(height: 17),
+                                  TextInputForm(
+                                    labelText:
+                                        S.of(context).confirmPasswordLabel,
+                                    hintText: S.of(context).passwordHintText,
+                                    isSensitive: true,
+                                    validator: ConfirmPasswordFieldValidator(
+                                      oldPassword: _password,
+                                    ),
+                                    onFieldSaved: (value) {},
+                                    onTaped: () {},
+                                  ),
+                                  const SizedBox(height: 17),
+                                  DropDownInputForm(
+                                    labelText: S.of(context).genderLabel,
+                                    hintText: S.of(context).genderHintText,
+                                    optionsList: const [
+                                      "Female",
+                                      "Male",
+                                      "Other",
+                                    ],
```

Comment:
WDYT about having this in a constants file for the options?

Reply:
You're right, I planned to changed this on the following PR 🙌